Sources/OktaIdx/IDXClient.swift

```diff
@@ -311,6 +318,33 @@ public final class IDXClient: NSObject {
 
         self.api.client = self
     }
+    
+    @objc(revokeToken:type:completion:)
+    public func revoke(token: Token, type: Token.RevokeType, completion: @escaping(_ successful: Bool, _ error: Error?) -> Void) {
+        let selectedToken: String?
```

Comment:
You may avoid optional. Just make `let selectedToken: String`.

Reply:
The `refreshToken` property may be nil, so this variable still needs to remain optional.